Disable the make check runtime tests, which cannot be run when
cross-compiled.

Upstream-Status: Pending

Signed-off-by: Scott Garman <scott.a.garman@intel.com>

diff -urN insserv-1.14.0.orig//Makefile insserv-1.14.0/Makefile
--- insserv-1.14.0.orig//Makefile	2010-11-28 17:12:01.715597293 -0800
+++ insserv-1.14.0/Makefile	2010-11-28 17:16:33.837780660 -0800
@@ -119,8 +119,8 @@
 ifeq ($(ISSUSE),-DSUSE)
 	issuse=true tests/common
 #	issuse=true tests/suse
-else
-	tests/common
+#else
+#	tests/common
 endif
 
 install:	$(TODO) check
